compiler/bindings/c/iree/compiler/dialects/iree_codegen.h

```diff
@@ -30,6 +31,26 @@ MLIR_CAPI_EXPORTED
 uint32_t
 ireeCodegenDispatchLoweringPassPipelineAttrGetValue(MlirAttribute attr);
 
+MLIR_CAPI_EXPORTED bool
+ireeAttributeIsACodegenTranslationInfoAttr(MlirAttribute attr);
+
+MLIR_CAPI_EXPORTED MlirTypeID ireeCodegenTranslationInfoAttrGetTypeID(void);
+
+struct ireeCodegenTranslationInfoParameters {
+  MlirAttribute passPipeline;      // DispatchLoweringPassPipelineAttr.
+  MlirAttribute codegenSpec;       // Optional SymbolRefAttr.
```

Comment:
Interesting. At some point we should make the codegenSpec part of the configuration. Not actionable for this pr though.